app/enterprise/1.5.x/kong-for-kubernetes/install.md

```diff
@@ -158,13 +152,223 @@ $ export PROXY_IP=$(kubectl get -o jsonpath="{.status.loadBalancer.ingress[0].ip
+        --docker-password=<your-bintray-api-key>
+    ```
+
+2. (Optional) Create a password for the super admin:
+    ```
+    oc create secret generic kong-enterprise-superuser-password -n kong --from-literal=password=<your-password>
+    ```
+    Though not required, this is recommended for users that wish to use RBAC, as it cannot be done after initial setup.
+{% endnavtab %}
+{% endnavtabs %}
+### Enable the sessions plugin for Manager and Dev Portal
+In the following steps, replace `<your-password>` with a secure password.
+
+{% navtabs %}
+{% navtab kubectl %}
+
+1. Create a sessions config file for Kong Manager:
+    ```
+    $ cat > admin_gui_session_conf
+    {"cookie_name":"admin_session","cookie_samesite":"off","secret":"<your-password>","cookie_secure":false,"storage":"kong"}
+    ```
+2. Create a sessions config file for Kong Dev Portal:
+    ```
+    $ cat > portal_session_conf
+    {"cookie_name":"portal_session","cookie_samesite":"off","secret":"<your-password>","cookie_secure":false,"storage":"kong"}
+    ```
+
+3. Create secret:
+    ```
+    $ kubectl create secret generic kong-session-config -n kong --from-file=admin_gui_session_conf --from-file=portal_session_conf
+    ```
+{% endnavtab %}
+{% navtab OpenShift oc %}
+
+1. Create a sessions config file for Kong Manager:
+    ```
+    $ cat > admin_gui_session_conf
+    {"cookie_name":"admin_session","cookie_samesite":"off","secret":"<your-password>","cookie_secure":false,"storage":"kong"}
+    ```
+2. Create a sessions config file for Kong Dev Portal:
+    ```
+    $ cat > portal_session_conf
+    {"cookie_name":"portal_session","cookie_samesite":"off","secret":"<your-password>","cookie_secure":false,"storage":"kong"}
+    ```
+
+3. Create secret:
+    ```
+    $ oc create secret generic kong-session-config -n kong --from-file=admin_gui_session_conf --from-file=portal_session_conf
+    ```
+{% endnavtab %}
+{% endnavtabs %}
+### Prepare the values.yaml
+
+1. Create a `values.yaml` file based on the sample in the [Kong charts repository](https://github.com/Kong/charts/blob/master/charts/kong/values.yaml). This file sets all the necessary parameters for your Kong environment.
+
+2. Minimally, for setting up Kong Enterprise on Kubernetes, you will need to set the following parameters:
+
+    Parameter | Value  
+    ----------|-------
+    `env.database` | `"postgres"` or `"cassandra"`
+    `env.password.valueFrom.secretKeyRef.name` | Name of secret that holds the super admin password. In the example above, this is set to `kong-enterprise-superuser-password`.
+    `env.password.valueFrom.secretKeyRef.key` | The type of secret key used for authentication. If you followed the default settings in the example above, this is `password`.
+    `image.repository` | The Docker repository. In this case, `kong-docker-kong-enterprise-edition-docker.bintray.io/kong-enterprise-edition`
+    `image.tag` | The Docker image tag you want to pull down, e.g. `"1.5.0.2-alpine"`
```

Comment:
Would `latest` be a better substitute here to avoid cases where this goes out of sync for newer versions in future?

Reply:
No--we shouldn't recommend `latest` since it might end up in actual configuration and result in unintentionally spawning a new version after a pod restart. Having a possibly outdated value in an example isn't a huge concern for me, though we should strive to keep actual defaults current.